source/manual/alerts/slo-rate-checks-account.html.md

```diff
@@ -0,0 +1,45 @@
+---
+owner_slack: "#govuk-account-tech"
+title: GOV.UK Account SLO error rate exceeded 1% in 10 minute window
+section: Icinga alerts
```

Comment:
Is this an Icinga alert, or do they stem from Prometheus?

Reply:
So... here's the config:
https://github.com/alphagov/govuk-puppet/pull/11171

This is my first venture into this stuff, but I'm pretty sure we're not touching prometheus.
My understanding is that it'd got:
Graphite  query -> Icinga warning?

Reply:
Yeah, these are Icinga  alerts